src/stories/StyledTab.tsx

```diff
@@ -0,0 +1,79 @@
+import { PropsWithChildren, ReactElement } from 'react';
+import DfxIcon, { IconColors, IconSizes, IconVariant } from './DfxIcon';
+import { TabType } from './StyledTabContainer';
+
+interface StyledTabProps extends PropsWithChildren {
+  tabId: number;
+  updateActiveTab: (tId: number) => any;
+  openTab: number;
```

Comment:
Consistent naming would make it easier to use, e.g. `activeTab` and `setActiveTab`

Reply:
Even simpler would be just `active` and `setActive`? The tab probably doesn't care about its ID 😉

Reply:
changed it.